src/app/modules/time-clock/components/entry-fields/entry-fields.component.spec.ts

```diff
@@ -134,6 +161,37 @@ describe('EntryFieldsComponent', () => {
     expect(toastrServiceStub.error).toHaveBeenCalled();
   });
 
+  it('displays error message when new hour entered is in the past of other entry', () => {
+    component.newData = entry;
+    component.activeEntry = entry ;
+    component.setDataToUpdate(entry);
+    spyOn(toastrServiceStub, 'error');
+
+    const hourInTheFuture = moment().add(-6, 'hour').format('HH:mm:ss');
```

Comment:
how the time can be in the future if you're actually subtracting hours. BTW, you can use the function `subtract` instead of add in this case

Reply:
thanks, I already change it